osdep/timer.c

```diff
@@ -73,11 +75,19 @@ int64_t mp_time_ns_add(int64_t time_ns, double timeout_sec)
     return time_ns + ti;
 }
 
-#if !HAVE_WIN32_INTERNAL_PTHREADS
 static int get_realtime(struct timespec *out_ts)
 {
 #if defined(_POSIX_TIMERS) && _POSIX_TIMERS > 0
     return clock_gettime(CLOCK_REALTIME, out_ts);
+#elif defined(_WIN32)
+    union {
+        FILETIME ft;
+        ULARGE_INTEGER i;
+    } r;
+    GetSystemTimePreciseAsFileTime(&r.ft);
+    out_ts->tv_sec = r.i.QuadPart / UINT64_C(10000000);
+    out_ts->tv_nsec = r.i.QuadPart % UINT64_C(10000000);
```

Comment:
```
out_ts->tv_nsec = r.i.QuadPart % UINT64_C(10000000) * 100;
```

Reply:
Also not sure if we want to adjust it to posix epoch? If we aim to have direct compatibility with it.

Reply:
oh right, MS uses a different epoch here. Well that's annoying.